packages/core/README.md

```diff
@@ -1,6 +1,39 @@
-# Theia is a cloud & desktop IDE framework implemented in TypeScript.
+# Theia - Core extension
 
-See [here](https://github.com/theia-ide/theia) for a detailed documentation.
+## Logging configuration
+
+It is possible to change the log level for the entire Theia application by
+passing it the `--log-level={fatal,error,warn,info,debug,trace}` option.  For
+more fine-grained adjustment, it is also possible to set the log level per
+logger (i.e. per topic).  The `root` logger is a special catch-all logger
+through which go all messages not sent through a particular logger.  To change
+the log level of particular loggers, create a config file such as
+
+```json
+{
+  "defaultLevel": "info",
+  "levels": {
+    "terminal": "debug",
+    "task": "error"
+  }
+}
+```
+
+where `levels` contains the logger to log level mapping.  `defaultLevel`
+contains the log level to use for loggers not specified in `levels`.  This file
+can then be specified using the `--log-config` option.  Theia will watch that
+file for changes, so it is possible to change log levels at runtime by
+modifying this file.
+
+It is unfortunately not possible currently to query Theia for the list of
```

Comment:
"possible currently" -> "currently possible"? or "not possible, currently, "? Not even sure it's actually better or if it just seems that way me.

Reply:
"currently not possible"?